### DISCUSSION TITLE
[FR] Suffix/Prefix on Summarized table columns

### DISCUSSION THREAD

**Discussion by rvzug** [filamentphp/filament#12762] [Ideas]
Is there a way to - or is this a nice feature request - to add prefix/suffix to table summarizers? 

![image](https://github.com/filamentphp/filament/assets/7152234/b78858b6-1832-4049-bd07-929d42894388)

We have a table combining percentage, as well as volume, as weight. Therefor we want to define the units inside each table cell, including the summarized row.

Adding ```->formatStateUsing(fn($state) => $state . ' xx')``` works, but is not a nice solution is you want to keep the state clean.

**Comment by rvzug** [comment]
And when using ```->formatStateUsing(fn($state) => $state . ' xx')``` you will lose things link the ```->numeric()``` functionallity.

**Comment by rvzug** [comment]
@danharrin I saw your tweet about adding feature-PRs for v3, I also saw this feature has not been added to v4 yet. I want to create a PR for this feature, should I add it in v3 or v4?

**Reply by danharrin** [reply]
Yes I would accept a PR for v3 as long as it is published today or tomorrow, so I have time to review before Monday

**Reply by rvzug** [reply]
Done in #13702
